test/conformance/cgroup_test.go

```diff
@@ -29,11 +30,15 @@ import (
 )
 
 const (
-	cpuLimit    = 1   // CPU
-	memoryLimit = 128 // MB
-	cpuRequest  = 1   // CPU
+	cpuLimit    = 1     // CPU
+	memoryLimit = 128   // MB
+	cpuRequest  = 0.125 // CPU
 )
 
+func toMilliValue(value float64) string {
+	return fmt.Sprintf("%dm", int(value*1000))
```

Comment:
At this point, why not just 
`const  (mcpuLimit = 1000, mcpuRequest=125) // m as in milli` ?
and then remove this math?

Reply:
Cause then I'll need to change the math below, that calculates the cpu.shares value to the inverse of this 🙂

Reply:
In the actual test below we do `x*1000` again, right?

Reply:
Nope, there's also a `* 1024` for the requests, which is where it gets a little more hairy.

Reply:
Ah, 1024 is basically 1000 ;-)